custom_components/tuya_local/devices/loratap_wifi_curtain_switch_double.yaml

```diff
@@ -0,0 +1,175 @@
+name: Loratap Wifi Double Curtain Switch
+products:
+  - id: zqypumy8ot4clzyi
+primary_entity:
+  entity: cover
+  name: Curtain
+  class: blind
+  dps:
+    - id: 1
+      name: control
+      type: string
+      mapping:
+        - dps_val: open
+          value: open
+        - dps_val: close
+          value: close
+        - dps_val: stop
+          value: stop
+    - id: 2
+      name: position
+      type: integer
+      unit: "%"
+      range:
+        max: 100
+        min: 0
+      mapping:
+        - step: 1
+secondary_entities:
+  - entity: cover
+    name: Curtain 2
+    dps:
+      - id: 4
+        name: control
+        type: string
+        mapping:
+          - dps_val: open
+            value: open
+          - dps_val: close
+            value: close
+          - dps_val: stop
+            value: stop
+      - id: 5
+        name: position
+        type: integer
+        unit: "%"
+        range:
+          max: 100
+          min: 0
+        mapping:
+          - step: 1
+  - entity: number
+    name: Position 1
+    category: config
+    icon: "mdi:blinds-horizontal-closed"
+    dps:
+      - id: 2
+        name: value
+        type: integer
+        unit: "%"
+        range:
+          max: 100
+          min: 0
+        mapping:
+          - scale: 0
+          - step: 1
+  - entity: number
+    name: Position 2
+    category: config
+    icon: "mdi:blinds-horizontal-closed"
+    dps:
+      - id: 5
+        name: value
+        type: integer
+        unit: "%"
+        range:
+          max: 100
+          min: 0
+        mapping:
+          - scale: 0
+          - step: 1
+  - entity: number
+    name: timer
+    category: config
+    icon: "mdi:timer"
+    dps:
+      - id: 10
+        name: value
+        type: integer
+        range:
+          max: 120
+          min: 2
+        mapping:
```

Comment:
There should be a unit here (min, or h?)  Or if the unit is s, this is probably not a simple timer, but the setting for how long the curtains take to open/close fully, which implies a different name for the entity.

And remove the mapping, because it is wrong as stated above, and should work OK with the defaults of no scaling and step of 1.